ext/lib/crypto/mbedtls/Kconfig.tls-generic

```diff
@@ -135,16 +181,45 @@ config TLS_CIPHER_DES_ENABLED
 	bool "Enable the DES block cipher"
 	default y
 
+config TLS_CIPHER_ARC4_ENABLED
+	bool "Enable the ARCFOUR stream cipher"
+
+config TLS_CIPHER_CHACHA20_ENABLED
+	bool "Enable the ChaCha20 stream cipher"
+
+config TLS_CIPHER_BLOWFISH_ENABLED
+	bool "Enable the Blowfish block cipher"
+
 config TLS_CIPHER_CCM_ENABLED
 	bool "Enable the Counter with CBC-MAC (CCM) mode for 128-bit block cipher"
 	depends on TLS_CIPHER_AES_ENABLED || TLS_CIPHER_CAMELLIA_ENABLED
 
+config TLS_CIPHER_MODE_XTS_ENABLED
+	bool "Enable Xor-encrypt-xor with ciphertext stealing mode (XTS) for AES"
```

Comment:
And here I'd say it goes just too detailed - for layman, both "XTS" and "Xor-encrypt-xor with ciphertext stealing" mean about the same - weird crypto mumbo-jumbo ;-). No need to change now that the work is done, the effort to educate people on spot is always appreciated of course.

Reply:
Just to clarify, this "mumbo-jumbo" is actually original description from mbedTLS config file template.